airbyte-integrations/connectors/source-sftp-bulk/source_sftp_bulk/client.py

```diff
@@ -168,15 +169,25 @@ def get_files(self, prefix, search_pattern=None, modified_since=None, most_recen
 
         return sorted_files
 
+    def peek_line(self, f):
+        pos = f.tell()
+        line = f.readline()
+        f.seek(pos)
+        return line
+
     @backoff.on_exception(backoff.expo, (socket.timeout), max_tries=5, factor=2)
-    def fetch_file(self, fn: Mapping[str, Any], file_type="csv") -> pd.DataFrame:
+    def fetch_file(self, fn: Mapping[str, Any], separator, file_type="csv") -> pd.DataFrame:
```

Comment:
Can you please add default value for the separator?
#Option 1 - Add "," as default separator value (as default file type is csv) 
#Option 2 - Add None as default separator value (but the existing pipelines which uses default values will incur Sniffer processing overhead.
So my suggestion is to use option 1 and add comments as passing None as separator will try to identify the separator using Sniffer.

Reply:
@anamargaridarl  the method definition is not updated. Can you please recheck it?

Reply:
`def fetch_file(self, fn: Mapping[str, Any], separator=",", file_type="csv")`